ui/src/app/pages/runtimes-list.tsx

```diff
@@ -51,7 +50,10 @@ const ajax = (signal) => {
     Metrics: { captureEvent: () => undefined },
     Disks: {
       disksV1: () => ({
-        list: () => jsonLeoFetch('/api/google/v1/disks?role=creator'),
+        list: () =>
+          jsonLeoFetch(
+            '/api/google/v1/disks?role=creator&includeLabels=saturnWorkspaceNamespace,saturnWorkspaceName'
```

Comment:
does it work if u don't have `includeLabels=saturnWorkspaceNamespace,saturnWorkspaceName`? 
just curious what the API behavior is :)

Reply:
It does not. Fetching the runtimes returns the labels without this parameter, but fetching disks does not.